orchagent/vxlanorch.cpp

```diff
@@ -713,7 +1341,7 @@ bool VxlanTunnelOrch::addOperation(const Request& request)
         return true;
     }
 
-    vxlan_tunnel_table_[tunnel_name] = std::unique_ptr<VxlanTunnel>(new VxlanTunnel(tunnel_name, src_ip, dst_ip));
+    vxlan_tunnel_table_[tunnel_name] = std::unique_ptr<VxlanTunnel>(new VxlanTunnel(tunnel_name, src_ip, dst_ip), TNL_CREATION_SRC_CLI);
```

Comment:
wrong bracket `new VxlanTunnel(tunnel_name, src_ip, dst_ip), TNL_CREATION_SRC_CLI)` should change to `new VxlanTunnel(tunnel_name, src_ip, dst_ip, TNL_CREATION_SRC_CLI))`

Reply:
Done